pages/dao/[symbol]/proposal/components/instructions/ProposalForm.tsx

```diff
@@ -0,0 +1,180 @@
+        return (
+          <CreateAssociatedTokenAccount
+            index={index}
+            governanceAccount={governanceAccount}
+          />
+        )
+      case Instructions.CreateSolendObligationAccount:
+        return (
+          <CreateObligationAccount
+            index={index}
+            governanceAccount={governanceAccount}
+          />
+        )
+      case Instructions.InitSolendObligationAccount:
+        return (
+          <InitObligationAccount
+            index={index}
+            governanceAccount={governanceAccount}
+          />
+        )
+      case Instructions.DepositReserveLiquidityAndObligationCollateral:
+        return (
+          <DepositReserveLiquidityAndObligationCollateral
+            index={index}
+            governanceAccount={governanceAccount}
+          />
+        )
+      case Instructions.RefreshSolendObligation:
+        return (
+          <RefreshObligation
+            index={index}
+            governanceAccount={governanceAccount}
+          />
+        )
+      case Instructions.RefreshSolendReserve:
+        return (
+          <RefreshReserve index={index} governanceAccount={governanceAccount} />
+        )
+      case Instructions.WithdrawObligationCollateralAndRedeemReserveLiquidity:
+        return (
+          <WithdrawObligationCollateralAndRedeemReserveLiquidity
+            index={index}
+            governanceAccount={governanceAccount}
+          />
+        )
+      case Instructions.AddLiquidityRaydium:
+        return (
+          <AddLiquidityToPoolRaydium
+            index={index}
+            governanceAccount={governanceAccount}
+          />
+        )
+      case Instructions.RemoveLiquidityRaydium:
+        return (
+          <RemoveLiquidityFromPoolRaydium
+            index={index}
+            governanceAccount={governanceAccount}
+          />
+        )
+      case Instructions.Mint:
+        return <Mint index={index} governance={governance}></Mint>
+      case Instructions.Base64:
+        return (
+          <CustomBase64 index={index} governance={governance}></CustomBase64>
+        )
+      case Instructions.None:
+        return <Empty index={index} governance={governance}></Empty>
+      case Instructions.MangoMakeChangeMaxAccounts:
+        return (
+          <MakeChangeMaxAccounts
+            index={index}
+            governance={governance}
+          ></MakeChangeMaxAccounts>
+        )
+      case Instructions.MangoChangeReferralFeeParams:
+        return (
+          <MakeChangeReferralFeeParams
+            index={index}
+            governance={governance}
+          ></MakeChangeReferralFeeParams>
+        )
+      case Instructions.Grant:
+        return <Grant index={index} governance={governance}></Grant>
+      case Instructions.Clawback:
+        return <Clawback index={index} governance={governance}></Clawback>
+      default:
+        null
+    }
+  }
+
+  return (
+    <>
+      {![
+        Instructions.Transfer,
```

Comment:
Look like we use the governance select for all implemented instructions so far? IMO we should remove this include for now